minos/common/configuration/config.py

```diff
@@ -46,6 +46,8 @@
     "commands.queue.password": "MINOS_COMMANDS_QUEUE_PASSWORD",
     "commands.broker": "MINOS_COMMANDS_BROKER",
     "commands.port": "MINOS_COMMANDS_PORT",
+    "saga.broker": "MINOS_SAGA_HOST",
```

Comment:
In order to follow the same naming convention: 
```suggestion
    "saga.broker": "MINOS_SAGA_BROKER",
```

Reply:
Done!